project/Related-projects.rst

```diff
@@ -35,6 +35,12 @@ Ubuntu Touch
 `Ubuntu Touch <https://ubports.com>`_ is based on Unity 8 and is developed by UBports. It is the continuation of the earlier Ubuntu Touch project from Canonical. UBports is actively working on making their future releases available on Halium.
 
 
+hybris-mobian
+-------------
+
+`hybris-mobian <https://hybris-mobian.org/>`_ is a GNU/Linux distribution based on top of Mobian, a Debian-based distribution for mobile devices.The goal of hybris-mobian is to be able to run Mobian on Android phones
```

Comment:
space before The
dot after phones

Reply:
@doniks The suggested changes have been committed & pushed.Please review  & merge the PR.